From 838d4b931cc1960a4bbaef3a50a8082883eeda06 Mon Sep 17 00:00:00 2001
From: "Chrissi^ (Chris Fiege)" <chris@tinyhost.de>
Date: Sat, 29 Jun 2019 23:41:07 +0200
Subject: [PATCH] doc/dev/package: Fix path to gluon.mk (#1774)

$(TOPDIR) does not contain a trailing slash.
Thus the 'gluon.mk'-include must include
the trailing slash. (Just like the link to
$(INCLUDE_DIR)/package.mk does.)

Signed-off-by: Chris Fiege <chris@tinyhost.de>
---
 docs/dev/packages.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/dev/packages.rst b/docs/dev/packages.rst
index 42173cec..a527aad4 100644
--- a/docs/dev/packages.rst
+++ b/docs/dev/packages.rst
@@ -10,7 +10,7 @@ Gluon package makefiles
 As many packages share the same or a similar structure, Gluon provides a ``package/gluon.mk`` that
 can be included for common definitions. This file replaces OpenWrt's ``$(INCLUDE_DIR)/package.mk``;
 it is usually included as ``include ../gluon.mk`` from Gluon core packages, or as
-``include $(TOPDIR)../package/gluon.mk`` from feeds.
+``include $(TOPDIR)/../package/gluon.mk`` from feeds.
 
 Provided macros
 ***************
-- 
2.17.1

